pypeit/data/standards/blackbody/make_spec.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from astropy import table
+import astropy.constants as cons
+import astropy.units as u
+
+
+def blackbody_func(a, teff, outname):
+    """
+    Generate a blackbody spectrum based on the normalisation and effective temperature.
+    See Suzuki & Fukugita, 2018, AJ, 156, 219:
+    https://ui.adsabs.harvard.edu/abs/2018AJ....156..219S/abstract
+
+    Args:
+        a (float):
+            flux normalisation factor
+        teff (float):
+            Effective temperature of the blackbody
+        outname (string):
+            The filename to write the spectrum to.
+
+    """
```

Comment:
Why are you writing these files out to disk and archiving them if you can generate the spectrum on the fly?

Reply:
Done - now generating these during runtime.